internal/configs/ingress_test.go

```diff
@@ -126,6 +126,41 @@ func TestPathOrDefaultReturnActual(t *testing.T) {
 	}
 }
 
+func TestGenerateIngressPath(t *testing.T) {
+	tests := []struct {
+		pathType v1beta1.PathType
+		path     string
+		expected string
+	}{
+		{
+			pathType: v1beta1.PathTypeExact,
+			path:     "/path/to/resource",
+			expected: "= /path/to/resource",
+		},
+		{
+			pathType: v1beta1.PathTypePrefix,
+			path:     "/path/to/resource",
+			expected: "/path/to/resource",
+		},
+		{
+			pathType: v1beta1.PathTypeImplementationSpecific,
+			path:     "/path/to/resource",
+			expected: "/path/to/resource",
+		},
+		{
+			pathType: v1beta1.PathType(""),
```

Comment:
is the case when pathType is an empty string possible? could you double check?

Reply:
i dont think this is possible